MatrixSDK/Utils/Media/MXMediaLoader.m

```diff
@@ -40,31 +40,22 @@
 
 NSString *const kMXMediaUploadIdPrefix = @"upload-";
 
-// Modified by Nedap. Store accessToken to set the authorization token on media requests (BER-229)
+
```

Comment:
Should this comment be removed?

Reply:
Yes, I added this comment for `accessToken` property and I removed it now because the same property (with the exact name) has been added by matrix 😄.  So, I kept matrix changes in this case.